src/activities/parse-docmap.test.ts

```diff
@@ -5,10 +5,65 @@ const simpleDocmap = `{
   "steps": []
 }`;
 
+const fullDocmap = `{
+  "@context": "https://w3id.org/docmaps/context.jsonld",
+  "type": "docmap",
+  "id": "http://mock-datahub/enhanced-preprints/docmaps/v1/by-publisher/elife/get-by-doi/10.1101%2F000001",
+  "created": "2022-11-11T05:02:51+00:00",
+  "updated": "2022-11-11T05:02:51+00:00",
+  "publisher": {
+    "account": {
+      "id": "https://sciety.org/groups/elife",
+      "service": "https://sciety.org"
+    },
+    "homepage": "https://elifesciences.org/",
+    "id": "https://elifesciences.org/",
+    "logo": "https://sciety.org/static/groups/elife--b560187e-f2fb-4ff9-a861-a204f3fc0fb0.png",
+    "name": "eLife"
+  },
+  "first-step": "_:b0",
+  "steps": {
+    "_:b0": {
+      "actions": [
+        {
+          "participants": [],
+          "outputs": [
+            {
+              "type": "preprint",
+              "doi": "10.1101/000001",
+              "url": "https://www.biorxiv.org/content/10.1101/000001v1",
+              "published": "2022-11-14",
+              "versionIdentifier": "1",
+              "_tdmPath": "s3://biorxiv/dummy-1.meca"
+            }
+          ]
+        }
+      ],
+      "assertions": [
+        {
+          "item": {
+            "type": "preprint",
+            "doi": "10.1101/000001",
+            "versionIdentifier": "1"
+          },
+          "status": "manuscript-published"
+        }
+      ],
+      "inputs": []
+    }
+  }
+}`;
+
 describe('parse-docmap-activity', () => {
-  it('parses docmap passed as param', async () => {
+  it('fails to parse a docmap that does not contain a tdmPath', async () => {
```

Comment:
parsing could "fail" for lots of reasons - this probably isn't the right description of this failure (there's nothing in simpleDocmap to parse).

Reply:
I'm thinking something instead like 'fails to parse an invalid docmap'